hello_world_java/README.txt

```diff
@@ -0,0 +1,3 @@
+HELLO WORLD IN JAVA
```

Comment:
move the file inside hello_world directory, if already java file is present then delete this file.